backend/.env.example

```
@@ -37,3 +37,11 @@ SOROBAN_RPC_URL=https://soroban-testnet.stellar.org:443
 # Additional Configuration
 # Add other configuration variables as needed
 ENVIRONMENT=development
+
+# Monitoring Configuration
+# Enable/disable metrics collection (true/false)
+METRICS_ENABLED=true
+# Prometheus metrics endpoint path
+METRICS_PATH=/metrics
+# HTTP metrics endpoint path (auto-enabled by actix-web-prom)
+HTTP_METRICS_PATH=/metrics/http
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HTTP_METRICS_PATH key should go before the METRICS_ENABLED key

(UnorderedKey)